types/src/deltaE/deltaEHCT.d.ts

```diff
@@ -0,0 +1,5 @@
+import Color, { ColorObject } from "../color.js";
+export default function (
```

Comment:
This should also be exported from `types/src/deltaE/index.d.ts` along with the other deltaE methods.

Reply:
Okay